packages/features/ee/teams/lib/payments.ts

```diff
@@ -211,17 +209,17 @@ export const purchaseTeamOrOrgSubscription = async (input: {
 
     log.debug("Getting price ID", safeStringify({ fixedPriceId, isOrg, teamId, pricePerSeat }));
 
-    if (!pricePerSeat) {
-      return fixedPriceId;
-    }
-
-    const priceObj = await stripe.prices.retrieve(fixedPriceId);
-    if (!priceObj) throw new Error(`No price found for ID ${fixedPriceId}`);
-
-    return priceObj;
+    return fixedPriceId;
   }
 };
 
+async function getCustomPrice(fixedPriceId: string) {
```

Comment:
Why are we calling it `getCustomPrice`? Custom Price terminology we are using currently to refer to a price created on demand.

`getPriceObject` could be a better name as it is just getting the stripe price object from the priceId.

Reply:
Also, the argument doesn't need to care if it is fixed, dynamic or custom. So, it could be named `priceId`.